packages/core/src/internal/io.js

```diff
@@ -143,9 +143,7 @@ export function call(fnDescriptor, ...args) {
     check(
       fnDescriptor,
       isNotDelayEffect,
-      `instead of writing \`yield call(delay, ${String(
-        args[0],
-      )})\` where delay is an effect from \`redux-saga/effects\` you should write \`yield delay(${String(args[0])})\``,
```

Comment:
`String(Object.create(null))` will throw an exception. So we could avoid stringifying args[0] here.

Reply:
its a dev path and it's strictly addressing the delay case, so maybe we could do this `typeof args[0] === 'number' ? args[0] : 'ms'`?